gpflow/kernels/mo_kernels.py

```diff
@@ -100,7 +110,8 @@ class SeparateIndependent(MultioutputKernel, Combination):
     - Independent: Latents are uncorrelated a priori.
     """
     def __init__(self, kernels, name=None):
-        Combination.__init__(self, kernels, name)
+        super().__init__(output_dim=len(kernels), kernels=kernels, name=name)
```

Comment:
pass through `**kwargs`

Reply:
we don't really use `name`... should we remove it?

Reply:
Given that `tf.Module` defines `name`, I think we should keep it to remain compatible (but we can replace it by pass-through of `**kwargs` here)

Reply:
yes i think we should keep, though sadly our own module does nothing with `name`, i think - though it should if it doesnt!

Reply:
well, the superclass tf.Module uses it